servers/rendering_server.cpp

```diff
@@ -2282,6 +2282,10 @@ RenderingServer::RenderingServer() {
 	GLOBAL_DEF_RST("rendering/textures/vram_compression/import_etc2", true);
 	GLOBAL_DEF_RST("rendering/textures/vram_compression/import_pvrtc", false);
 
+	GLOBAL_DEF("rendering/textures/lossless_compression/force_png", false);
+	GLOBAL_DEF("rendering/textures/lossless_compression/webp_compression_level", 2);
+	ProjectSettings::get_singleton()->set_custom_property_info("rendering/textures/lossless_compression/webp_compression_level", PropertyInfo(Variant::INT, "", PROPERTY_HINT_RANGE, "0,9,1"));
```

Comment:
This should repeat the name in the `PropertyInfo` (at least that's what is done in the rest of engine code):
```suggestion
	ProjectSettings::get_singleton()->set_custom_property_info("rendering/textures/lossless_compression/webp_compression_level", PropertyInfo(Variant::INT, "rendering/textures/lossless_compression/webp_compression_level", PROPERTY_HINT_RANGE, "0,9,1"));
```

Reply:
I would argue that it should be removed everywhere else instead.
The second parameter of the PropertyInfo ctor sets the *name* field. This field is then overwritten in *set_custom_property_info*.

It is redundant, but I can change it if you insist.

Reply:
I agree that it could be removed everywhere, but that should be done in a different PR/proposal. Current uses all repeat the name, so I prefer to keep doing it here to be consistent, and we can change everything once we decide how to fix it (either by using `""` everywhere, or by implementing a slightly different API).

(See output of `rg set_custom_property_info.*PropertyInfo`.)